test/unit/lib/plugins/aws/utils/formatLambdaLogEvent.test.js

```diff
@@ -2,30 +2,32 @@
 
 const expect = require('chai').expect;
 const dayjs = require('dayjs');
-const chalk = require('chalk');
-const os = require('os');
+const { style } = require('@serverless/utils/log');
 const formatLambdaLogEvent = require('../../../../../../lib/plugins/aws/utils/formatLambdaLogEvent');
 
 describe('#formatLambdaLogEvent()', () => {
   it('should format invocation report', () => {
     const msg =
-      'REPORT\tRequestId: 99c30000-b01a-11e5-93f7-b8e85631a00e\tDuration: 0.40 ms\tBilled Duration: 100 ms\tMemory Size: 512 MB\tMax Memory Used: 30 MB';
+      'REPORT RequestId: 99c30000-b01a-11e5-93f7-b8e85631a00e\tDuration: 0.40 ms\tBilled Duration: 100 ms\tMemory Size: 512 MB\tMax Memory Used: 30 MB';
+    const expectedMsg = style.aside(
+      'END Duration: 0.40 ms\tBilled Duration: 100 ms\tMemory Size: 512 MB\tMax Memory Used: 30 MB'
+    );
 
-    expect(formatLambdaLogEvent(msg)).to.equal(chalk.grey(msg + os.EOL));
+    expect(formatLambdaLogEvent(msg)).to.deep.equal(expectedMsg);
```

Comment:
IIRC we're compring here strings, why `.deep` (this feels dedicated for objects)?

Reply:
The result here is an array due to how `style` wrappers work

Reply:
`style` decorators always return strings. They just support array input.